src/Services/InspectionsApi.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Dxw\Whippet\Services;
+
+// Responsible for calling an API and returning information about dxw inspections
+// associated with a plugin
+class InspectionsApi
+{
+    public function __construct(
+        $host,
+        \Dxw\Whippet\Services\JsonApi $json_api
+    ) {
+        $this->host = $host;
+        $this->jsonApi = $json_api;
+    }
+
+    public function get_inspections($plugin_slug)
+    {
+        $result = $this->jsonApi->get($this->url($plugin_slug));
+        if ($result->isErr()) {
+            return $result;
+        }
+        $inspections = $this->buildInspections($result->unwrap());
+        return \Result\Result::ok($inspections);
+    }
+
+    private function buildInspections($raw_inspections)
+    {
+        return array_map(function ($raw_inspection) {
+            return new Inspection(
+                $raw_inspection['date'],
+                $raw_inspection['versions'],
+                $raw_inspection['result'],
+                $raw_inspection['url']
+            );
+        }, $raw_inspections);
+    }
+
+    private function url($plugin_slug)
+    {
+        return $this->host.'/wp-json/v1/inspections/'.$plugin_slug;
```

Comment:
I'd be tempted to remove all the url-related variables (both here and the `$inspections_api_host` defined in Dependencies.php), and define them all in one place - preferably a config file of some sort (maybe whippet.json?). That'd make them easier to manage, and avoid the risk of tests/functionality being broken by a url change.